### ISSUE TITLE
Improve GH Pages

### ISSUE THREAD

**Issue by jasonLaster** [devtools-html/debugger.html#1613]
I added a theme for the debugger.html to see what it would be like and it's pretty cool:

https://devtools-html.github.io/debugger.html/videos.html

Having a nice website experience helps us in a couple ways:
* It is a better first impression
* It makes the content more readable

**Tasks**
- [x] the parser is missing some things
- [x] we currently index on the docs repo, but I think we should move that to root for the project README.md

**Parser**

the markdown is not the same and is somewhat lacking:

* https://devtools-html.github.io/debugger.html/videos.html
* https://devtools-html.github.io/debugger.html/getting-setup.html

Part of the problem might be that the project doesn't have a [config.yml](https://github.com/devtools-html/debugger-examples/blob/master/_config.yml) like the debugger-examples. 

Part of the solution is likely the parser: https://github.com/blog/2100-github-pages-now-faster-and-simpler-with-jekyll-3-0

Lastly, this is not really a priority at the moment. 

CC @clarkbw

**Comment by clarkbw** [comment]
> we currently index on the docs repo, but I think we should move that to root for the project README.md

Yeah, especially if we linked to the various docs from the README this would make sense to me.

**Comment by jasonLaster** [comment]
I changed GH Pages directory to be master

**Comment by jasonLaster** [comment]
If somebody has **GH Pages** or **Jekyll** experience and would be able to help us improve this, it would be really appreciated.

**Comment by clarkbw** [comment]
Could be looking a lot better! 😼 https://devtools-html.github.io/debugger.html/

**Comment by jasonLaster** [comment]
@clarkbw looking better:
https://devtools-html.github.io/debugger.html/

all the links are broken tho 😈

**Comment by clarkbw** [comment]
Hmm, not sure how we have links that work in HTML and the README at the same time.  In HTML you want to link to [https://github.com/devtools-html/debugger.html/blob/master/docs/debugging-the-debugger](https://github.com/devtools-html/debugger.html/blob/master/docs/debugging-the-debugger.md) but in Markdown you need to link to [https://github.com/devtools-html/debugger.html/blob/master/docs/debugging-the-debugger](https://github.com/devtools-html/debugger.html/blob/master/docs/debugging-the-debugger.md)**.md**

**Comment by irfanhudda** [comment]
We could use [https://devtools-html.github.io/](https://devtools-html.github.io/) which will require a separate repository to host website, docs and other related things. It could also have documentation for other packages in `devtools-core` and of `devtools-reps` in single repository. But I'm not sure how feasible this would be.

**Comment by jasonLaster** [comment]
Good call @irfanhudda I'm sure at some point we'll want that to describe the larger project!

**Comment by jasonLaster** [comment]
This looks helpful
https://github.com/github/markup/issues/900

**Comment by jasonLaster** [comment]
Hmm, #1706 didn't quite do it.

Here's a way to set up gh pages locally which could help
https://help.github.com/articles/setting-up-your-github-pages-site-locally-with-jekyll/


With this said, I'm okay waiting on this for now as it's a low priority compared to the developer workflow imo

**Comment by liorpr** [comment]
I can help, but not sure how to approach this, where's the website repo? what are the changes need to be made?

**Comment by irfanhudda** [comment]
@liorpr At this time [https://devtools-html.github.io/debugger.html/](https://devtools-html.github.io/debugger.html/) is generated by github using this projects' `README.md`.

The problem with this approach is that links (for other articles in `docs/` folder) are not generated properly. They point to raw markdown files.

For now, it would be good to have them point to actual html pages. Jekyll appears to be a good option as github pages uses it.

**Comment by liorpr** [comment]
Are there any mock ups/designs/something for this?

**Comment by jasonLaster** [comment]
haha - i think we just want it to be functional right now

**Comment by jasonLaster** [comment]
lets add proselint

https://github.com/webpack/webpack.js.org/blob/master/.proselintrc

and other things from webpack.js.org

**Comment by jasonLaster** [comment]
@irfanhudda you did an amazing job here!
